src/node/rpc/memberfrontend.h

```diff
@@ -301,6 +301,14 @@ namespace ccf
         // record vote
         proposal->votes[args.caller_id] = vote.ballot;
         proposals->put(vote.id, *proposal);
+
+        // record signed vote request
+        if (!args.signed_request.sig.empty())
```

Comment:
Should the command fail if there is not signature for this request?

Reply:
We do not enforce this for any of the other RPC requests, maybe we should discuss if there should be a list of request types that are required to be signed.